lib/msf/ui/web/console.rb

```diff
@@ -58,9 +58,9 @@ def initialize(framework, console_id, opts={})
     end
 
     # Initialize the console with our pipe
-    self.console = Msf::Ui::Console::Driver.new(
-      'msf',
-      '>',
+    web_prompt = opts['PROMPT'] || 'web_msf'
```

Comment:
The default should really stay as `'msf'`.

Reply:
Marking this as resolved as this change was reverted with https://github.com/rapid7/metasploit-framework/pull/13489/commits/4c32d3f34e61f91819ea80aa958c5e9e3e8ee887